src/Controls/tests/DeviceTests/Elements/CollectionView/CollectionViewTests.cs

```diff
@@ -403,6 +409,13 @@ await CreateHandlerAndAddToWindow<LayoutHandler>(layout, async handler =>
 						? containerHeight
 						: Math.Min(itemsCount * templateHeight, containerHeight);
 
+#if IOS
```

Comment:
Should this work on Windows and Android? 

If it currently doesn't and should, would it make sense to open an issue?

Reply:
I'd say: it should work on all platforms.
But apparently it only makes an actual difference on iOS (issue author is only talking about such platform).
I can try to make it work on Android, but I'll need some help for the Windows platform.

Reply:
More just curious if we should log an issue

If this test currently fails for Android/Windows we can just log an issue saying so and link to it. No need to hold this PR up for Android/Windows.

Reply:
Sounds good, I'll file a new issue then.
(Test is failing there)

Reply:
Done #24672